typescript-promises/main.ts

```diff
@@ -0,0 +1,11 @@
+import { takeAChance } from './take-a-chance.js';
+
+const testPromise = takeAChance('Kyle');
+
+testPromise.then((message) => {
+  console.log(message);
+});
+
+testPromise.catch((error) => {
+  console.log(error.message);
+});
```

Comment:
Works, but you usually chain them together
```suggestion
testPromise
  .then((message) => {
    console.log(message);
  })
  .catch((error) => {
    console.log(error.message);
  });
```